test/e2e/content/case.turned-off-for-host.test.js

```diff
@@ -0,0 +1,141 @@
+/*eslint no-unused-expressions: "off" */
+/*globals describe, before, after, it, expect, browser, storage, selectors, testURL, popupURL, optionsURL */
+
+describe("When the user has turned off the extension for this host, the page", function() {
+    let page;
+    let popup;
+    let options;
+    const text =
+        "On the driveway, she washed her motorcycle while he washed his car.";
+
+    describe("When the user has not yet reloaded the page, it", function() {
+        before(async function() {
+            options = await browser.newPage();
+            await options.goto(optionsURL);
+            page = await browser.newPage();
+            await page.goto(testURL + text);
+            popup = await browser.newPage();
+
+            // Turn off replacements on this host only
+            await storage.set(options, {
+                doNotReplaceList: [new URL(testURL).host]
+            });
+
+            // Open the popup, but don't yet reload the page
+            await popup.goto(popupURL);
+        });
+
+        after(async function() {
+            await storage.clear(options);
+
+            await popup.close();
+            await page.close();
+            await options.close();
+        });
+
+        it("should not yet change the status message", async function() {
+            const statusText = await popup.$eval(
+                selectors.status,
+                e => e.innerText
+            );
+
+            // The status message remains the same
+            expect(statusText).to.equal(
+                "Degender the Web has replaced gender pronouns on this page."
+            );
+        });
+
+        it("should prompt the user to reload", async function() {
+            const reloadText = await popup.$eval(
+                selectors.reloadMessage,
+                e => e.innerText
+            );
+
+            expect(reloadText).to.equal(
+                "Reload the page to revert pronoun replacements."
+            );
+        });
+
+        it("should show the 'Show changes' checkbox", async function() {
+            await popup.waitForSelector(selectors.showChangesCheckbox, {
+                visible: false
+            });
+        });
+
+        it("should not show the 'Show highlights' checkbox", async function() {
+            await popup.waitForSelector(selectors.showHighlightsCheckbox, {
+                visible: false
+            });
+        });
+
+        it("should show the 'Reload page' button", async function() {
+            await popup.waitForSelector(selectors.reloadPage, {
+                hidden: false
+            });
+        });
+    });
+
```

Comment:
Correct the test descriptions for the three tests above

Reply:
(and make sure the tests are correct!)

Reply:
Got it! I consistently use `visible` instead of `hidden` now too.